connect/transforms/src/test/java/org/apache/kafka/connect/transforms/ExtractFieldTest.java

```diff
@@ -111,7 +157,24 @@ public void nonExistentFieldWithSchemaShouldFail() {
             xform.apply(record);
             fail("Expected exception wasn't raised");
         } catch (IllegalArgumentException iae) {
-            assertEquals("Unknown field: nonexistent", iae.getMessage());
+            assertEquals("Unknown field: SingleFieldPath{version=V1, path=nonexistent}", iae.getMessage());
+        }
+    }
+
+    @Test
+    public void nonExistentNestedFieldWithSchemaShouldFail() {
+        xform.configure(Collections.singletonMap("field", "magic.nonexistent"));
+
+        final Schema fooSchema = SchemaBuilder.struct().field("foo", Schema.INT32_SCHEMA).build();
+        final Schema keySchema = SchemaBuilder.struct().field("magic", fooSchema).build();
+        final Struct key = new Struct(keySchema).put("magic", new Struct(fooSchema).put("foo", 42));
+        final SinkRecord record = new SinkRecord("test", 0, keySchema, key, null, null, 0);
+
+        try {
+            xform.apply(record);
+            fail("Expected exception wasn't raised");
+        } catch (IllegalArgumentException iae) {
+            assertEquals("Unknown field: SingleFieldPath{version=V1, path=magic.nonexistent}", iae.getMessage());
```

Comment:
Like with `nonExistentFieldWithSchemaShouldFail`, I don't think the field syntax version needs to be included in the error message. But if it is included, shouldn't it be V2 here instead of V1?